content/en/service-mesh.md

```diff
@@ -26,3 +26,16 @@ Service meshes add reliability, observability, and security features
 uniformly across all services across a cluster without requiring code changes. 
 Before service meshes, that functionality had to be encoded into every single service, 
 becoming a potential source of bugs and technical debt.
+
+In the [Sidecar Container](/sidecar-container/) model, each microservice pod is paired with its own proxy.
+This per-service proxy intercepts L3-L7 traffic, enforcing load-balancing, mutual TLS, 
+tracing and metrics on a fine-grained basis.
```

Comment:
@Sonichigo, I think we can simplify this. I'm not sure I'd expect a general glossary reader to know about L3-L7 traffic, or mutual TLS.

Reply:
Yeah, I agree. This is too complicated IMHO. Also, I would vote for a different line beginning. We using the same start of the sentence as the 2 sentences before.

Reply:
> Yeah, I agree. This is too complicated IMHO. Also, I would vote for a different line beginning. We using the same start of the sentence as the 2 sentences before.

I have updated the content from line +32 to +36. Just to be sure, you mean the intro sentence which is "This per-service proxy intercepts and manages network traffic,"

Reply:
I think if you remove the redundant part it should be fine :)